src/components/asset-list/RecyclerAssetList2/core/useMemoBriefSectionData.ts

```diff
@@ -28,30 +28,41 @@ const FILTER_TYPES = {
 export default function useMemoBriefSectionData({
   externalAddress,
   type,
-}: { externalAddress?: string; type?: AssetListType } = {}) {
-  const { briefSectionsData }: { briefSectionsData: any[] } = externalAddress
-    ? // `externalAddress` is a static prop, so hooks will always execute in order.
-      // eslint-disable-next-line react-hooks/rules-of-hooks
-      useExternalWalletSectionsData({ address: externalAddress })
-    : // eslint-disable-next-line react-hooks/rules-of-hooks
-      useWalletSectionsData();
-  const { isSmallBalancesOpen, stagger } = useOpenSmallBalances();
+  briefSectionsData,
+}: {
+  externalAddress?: string;
+  type?: AssetListType;
+  briefSectionsData?: any[];
+} = {}) {
+  let sectionsDataToUse: any[];
+
+  if (externalAddress) {
+    // `externalAddress` is a static prop, so hooks will always execute in order.
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    sectionsDataToUse = useExternalWalletSectionsData({
+      address: externalAddress,
+    }).briefSectionsData;
+  } else {
+    sectionsDataToUse = briefSectionsData!;
+  }
+
+  const { isSmallBalancesOpen } = useOpenSmallBalances();
   const { isSavingsOpen } = useOpenSavings();
   const { isInvestmentCardsOpen } = useOpenInvestmentCards();
   const { isCoinListEdited } = useCoinListEdited();
-  const { hiddenCoins } = useCoinListEditOptions();
+  const { hiddenCoinsObj } = useCoinListEditOptions();
   const { openFamilies } = useOpenFamilies();
 
-  const result = useDeepCompareMemo(() => {
+  const result = useMemo(() => {
     let afterDivider = false;
     let isGroupOpen = true;
     const stickyHeaders = [];
     let index = 0;
     let afterCoins = false;
     // load firstly 12, then the rest after 1 sec
-    let numberOfSmallBalancesAllowed = stagger ? 12 : briefSectionsData.length;
+    let numberOfSmallBalancesAllowed = sectionsDataToUse.length;
```

Comment:
do you intentionally remove staggering? If yes, maybe worth doing this in other places too?

Reply:
we don't have any other places that have stagger afaik